web/styles/dark_theme.css

```diff
@@ -58,6 +58,10 @@
         }
     }
 
+    kbd {
+        text-shadow: none;
```

Comment:
Previously, there was no specific styling for `kbd` elements in `dark_theme.css`; instead, styles were applied to individual classes wherever `kbd` elements were used. However, the property text-shadow: none is common to all `kbd` elements in dark mode. For the `#message-formatting` and `#keyboard-shortcuts` sections, we want the `kbd` elements to be styled more like `span.tooltip-hotkey-hint`, rather than following the usual styling for `kbd` elements in the dark theme.